sources/csharp2/KalturaClient/Request/BaseRequestBuilder.cs

```diff
@@ -75,231 +65,231 @@ public BaseRequestBuilder<T> SetCompletion(OnErrorHandler errorHandler)
             return this;
         }
 
+        public virtual Params getParameters(bool includeServiceAndAction)
+        {
+            Params kparams = new Params();
+
+            if (client != null)
+                kparams.Add(client.RequestConfiguration.ToParams(false));
+
+            kparams.Add(base.ToParams(false));
+
+            if (includeServiceAndAction)
+                kparams.Add("service", service);
```

Comment:
I don't see that you included the action.

Reply:
this method did not change from the original and was used as is .. 
it is a virtual method so i assume the concrete requests will override or add more logic to it.

please see original method: 
https://github.com/kaltura/clients-generator/blob/b201140892d8191aad4477b8a769ee273ba7cca2/sources/csharp2/KalturaClient/Request/BaseRequestBuilder.cs#L394